### DISCUSSION TITLE
RequestId has invalid data in Azure Functions

### DISCUSSION THREAD

**Discussion by zajda20** [getsentry/sentry-dotnet#897] [Q&A]
### Environment

How do you use Sentry?
self-hosted/on-premise

Which SDK and version?
.NET Sentry.Extensions.Logging 2.1.6

### Steps to Reproduce

I use azure functions and added sentry this way
```
services.AddLogging(lb =>
            {
                lb.AddSerilog(serilog);
                lb.AddSentry(o =>
                {
                    o.Dsn = --------;
                    o.MinimumEventLevel = ------------;
                    o.Release = ---------;
                    o.ServerName = ----------;
                    o.Environment = --------------;
                });
            });
```

### Expected Result

Because the function is triggered by HTTP requests I would expect tag RequestId to work properly.

### Actual Result

Among other tags, I have this 
![image](https://user-images.githubusercontent.com/26905762/106746825-59b73700-6623-11eb-9f4a-c56a502a9de7.png)

**Comment by bruno-garcia** [comment]
Seems like the broken tag has `{` in it. RequestId doesn't really have that. Maybe something else going on in there?
